app/controllers/spectator_sport/dashboard/session_windows_controller.rb

```diff
@@ -0,0 +1,17 @@
+module SpectatorSport
+  module Dashboard
+    class SessionWindowsController < ApplicationController
+      def show
+        @session_window = SessionWindow.find(params[:id])
+      end
+
+      def destroy
+        @session_window = SessionWindow.find(params[:id])
+        @session_window.events.delete_all
+        @session_window.delete
+
+        redirect_to "/spectator_sport_dashboard"
```

Comment:
I'm not happy with this, but `spectator_sport_dashboard_path` didn't work. I probably don't completely understand routing in context of Engines, so I didn't find a way to not use absolute path (which is not great).
Maybe you know how to improve this, @bensheldon ?

Reply:
Thanks for flagging that, I totally overlooked that. As you're probably aware, it's not great because the routes could be mounted at a different path by the parent application. 

I also struggled with some of the routing stuff. I think the problem may be that Engines are not intended to be namespace nested. This is the first gem I've done with 2 engines, which I believe is necessary to mount the client-api separately from the dashboard. I'll poke it some more and figure out a way forward.

Reply:
oh! I think it's simply because each engine is separately loading routes.rb, so it loads twice. Lemme fix that.

Reply:
Fixed it, I think 😓 https://github.com/bensheldon/spectator_sport/pull/8